### PR TITLE
Update kit logos

### PR DESCRIPTION
**Pull request by Grand-Thibault**
## Description
- all KIT logos and gallery banners are available as svg
- all references are updated

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by jSchuetz88 — CHANGES_REQUESTED** [review]
Looks great, but you missed a setting for the connector image: 
image

**Review by matbmoser — APPROVED** [review]

**Review by matbmoser — CHANGES_REQUESTED** [review]
Make sure there is no svg water mark. This usually happens when it was moved to PowerPoint

**Comment by Grand-Thibault** [comment]
> Looks great, but you missed a setting for the connector image: image

I fixed it, thx

**Comment by Grand-Thibault** [comment]
> Make sure there is no svg water mark. This usually happens when it was moved to PowerPoint

I moved it to Powerpoint --> no watermark --> all fine

**Comment by Grand-Thibault** [comment]
@matbmoser @jSchuetz88 @stephanbcbauer  I just added kit-colors.md. Please review it as well.

**Comment by ClosedSourcerer** [comment]
@Grand-Thibault 

Could you provide me with the  source for one or two of the images please?
I would like to try to generate the SVGs myself.
Some of the files came from draw.io appearently, while for others I was not able to determine the source.

I randomly checked 10 of the newly added .svg files and noticed the following:

- The file size seems way too big for truely optimized vector graphics or is at least inconsistent.

This lead me to have an indepth look into `static\img\kits\agents\agents-kit-gallery.drawio.svg` 
- SVG with embedded Base64 encoded image data.
- Decoding the base64 leads me to believe, that generated SVGs are way too overcomplicated and contain too much data.

Funnily enough:
- demand-and-capacity-management-kit-logo.drawio.svg is only 10KB in size.
- data-chain-kit-logo.drawio.svg is 216 KB in size. for pretty much the same picture:

image

BUT demand-and-capacity-management-kit-logo.drawio.svg is an svg with embedded PNG data. So at least partially it is not true vector graphic.

- Embedding a whole font into most of the SVGs (Especially the ones that do not contain any text) seems to also have contributed to the filesize.

File: static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
...QKnb9l/HcrP27YCPT/S/tyI5qpVXPppppGwmO6rImxQjauWWn9j6sslS9mNWC84bFQ3/pL8wnaatkae1HvY2O7QICJtYytl4uLexf7y3bLZHNR1mQDRpqPF8i/2HioZ34IezX24ui0TeDgNzRfHuN66ZQX6o1YDOfP2q3WWxlbODHoZG4ewnYWEb9vlCUpHuuUPSAXN37YUkUsqZUivdU6FG6sXup5YsV0optdO92nnhofEbxd9wlnKTSxX1YFEtprf/tYA48u13ugvV3cjyzQIRZ3lU626qlK6CliXVvyaTwbqDOy7W6mno9BuRwa57q+jTuPJ3I4NV9zKUcKpkV9bQPuhiqSTW46Abiyi7dbSUlOr1rf33nT+js/rkhJH6ePdplt06HWCPpur4Vle033X8ikusD2/cZ71onJlEavOmfXbqSRU71OvgDWe+7FOR/7JSN/lft9qmf2WiiZ4qJeVf1Rk/cetv095F67G6L6e/Kxgls0F8+8iVyldUvqRe7J6ywd2wi7XWh2JRvcBJnYb8DT9XKt99v1hRw1s8LBe9UoH8fug06MXzzChSp29tuMyPJiuL0S5orZCfd3u1aimQp6NM0h0zTcrF29urrfQ5+3sBMd19eYot+5SLvprLV40zXqsjFl+0pSV+j+uId42RtOfqiLixY9h3dUSqGm5nJt6/W7VouAfkt1cuaLjpsu15O+ILSOsXz9nXWNmi/m6vxGS7hturGKiZb63ajAuvnuUwA/kLMG70Tt34LYwbvTOtCQgxIPtOVKPOmOx+U6MOJs8XaF4dvHdRUDO7DUlWAX9mYek84OMdjPx7wJaX8fDZA3ayHR7AJdjJdr7XTA3cOzJwySXMgOwEGRbBvXeB0eElzKMFVzKBlEkjUE4uYYIdPF5VwuRYqCYb3O/oeLDdwKRYcDolTCAIp1MSu+cVkk+D23qKQrvBdTrIzA2M04FHLmGcjvfNTBVubwFyT4ObQeDVKoObmSMxN7jpFY4rA/NeO9q8DKSIBb5eGdzEHGe5wU0g0AJncM9yRRbAvOJwIgsQY5KkAuIY3EBHkgUweTnEMYQkScWWLADiNBGTJBZkAVz+QAbA5Q9kAFg5AAAAAAAgR4h/hXHlLYGc3nQAAAAASUVORK5CYII=" preserveAspectRatio="none" pointer-events="none"/>
```
Comment (ClosedSourcerer) [inline static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg:22]: This SVG contains embedded PNG Image data, that by its nature does not scale, making it no true vector graphics.
Also this specific file seems to have been generated differently from the rest.

Would you kindly ensure consistency over all logos and gallery drawing, please.

**Review by ClosedSourcerer — CHANGES_REQUESTED** [review]

File: static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg (hunk 1 above)
Comment (Grand-Thibault) [reply static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg:22]: Good point. I guess some logos were exported as png from figma where the kit icons were originally created. Will have a look at it.

**Comment by ClosedSourcerer** [comment]
Regarding the newly added: demand-and-capacity-management-kit-logo.drawio.svg in https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/1113/commits/92d152632c935c6785d6b0d163be45b110db37b0 

That file ist at least two-thirds data-garbage.

I made a quick manual edit to it.

The file started out at 30KB:  
demand-and-capacity-management-kit-logo drawio


I added roughly 3kb data by adding human readable formatting.
And then removed 19 KB by throwing out alot (not even all) of the garbage: 
CLEANED

**Review by jSchuetz88 — APPROVED** [review]
Looks good from my side now. Build and tested it.

**Comment by jSchuetz88** [comment]
> Regarding the newly added: demand-and-capacity-management-kit-logo.drawio.svg in [92d1526](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/commit/92d152632c935c6785d6b0d163be45b110db37b0)
> 
> That file ist at least two-thirds data-garbage.
> 
> I made a quick manual edit to it.
> 
> The file started out at 30KB: demand-and-capacity-management-kit-logo drawio
> 
> I added roughly 3kb data by adding human readable formatting. And then removed 19 KB by throwing out alot (not even all) of the garbage: CLEANED
> 
> The garbage mainly consist of `data-cell-id` attributes, which according to [SVG2 Spec](https://www.w3.org/TR/SVG2/struct.html#DataAttributes) may exist as custom attributes but they serve no purpose in the file. At least, no purpose that benefits the tractus-x homepage.

From my perspective, regardless of whether you are right, I don't see any reason why we should discuss the generation of .svg files by .drawio (or any other established software) here. If we expect this from our contributors to check and fix on the data structure level, we most likely will never see a contribution again. This is an improvement compared to the previous state and it works fine and looks good.

**Review by matbmoser — APPROVED** [review]
No more discussions. Please merge ASAP. 

Thank you @Grand-Thibault for your wonderful work!

File: static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg (hunk 1 above)
Comment (matbmoser) [reply static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg]: closing this thread, as I saw that @Grand-Thibault fixed it.

**Comment by matbmoser** [comment]
dismissing review from @ClosedSourcerer since the changes were included by @Grand-Thibault! Thx guys.

**Comment by matbmoser** [comment]
I am merging this ASAP

**Pull request merged**